src/persist-client/src/read.rs

```diff
@@ -215,13 +214,6 @@ where
     }
 
     /// Politely expires this subscribe, releasing its lease.
-    ///
-    /// There is a best-effort impl in Drop to expire a listen that wasn't
-    /// explicitly expired with this method. When possible, explicit expiry is
-    /// still preferred because the Drop one is best effort and is dependant on
-    /// a tokio [Handle] being available in the TLC at the time of drop (which
-    /// is a bit subtle). Also, explicit expiry allows for control over when it
-    /// happens.
```

Comment:
As far as I can tell, neither `Listen` not `Subscribe` implement `Drop`.

Reply:
IIUC the way this is meant to work is that `Listen` has a `ReadHandle`, and the `ReadHandle` implements `Drop`. So IIUC the gist of these is correct but vague... we could probably switch to mentioning the read handle explicitly.

(OTOH, the "These are only here" comments do just seem wrong... lots else going on.

Reply:
> IIUC the way this is meant to work is that `Listen` has a `ReadHandle`, and the `ReadHandle` implements `Drop`. So IIUC the gist of these is correct but vague... we could probably switch to mentioning the read handle explicitly.

Ah ok, that makes much more sense. I reverted the comments and just added some clarification to them. Will the `Drop` impl for `ReadHandle` even take care of returning the leased parts of a `Subscribe`?

```
        if let Some(parts) = self.snapshot.take() {
            for part in parts {
                self.return_leased_part(part)
            }
        }
```